packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -139,10 +140,13 @@ export const handleInput = assign({
 });
 
 export const handleSubmit = assign({
-  formValues: (context, event: AuthEvent) => ({
-    ...context['formValues'],
-    ...event.data,
-  }),
+  formValues: (context, event: AuthEvent) => {
+    const formValues = {
+      ...context['formValues'],
+      ...event.data,
+    };
+    return trimValues(formValues, 'password'); // do not trim password
```

Comment:
I'm assuming this doesn't need to exclude 'confirm_password' since that's not submitted?

Reply:
Yes, exactly, `confirm_password` is only for UI validation purposes and gets excluded. Do you think this is comment worthy?